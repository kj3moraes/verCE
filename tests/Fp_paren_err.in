def why(f) f + 1 );
