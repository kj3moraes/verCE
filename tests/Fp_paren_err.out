DEF:def 
ID:why 
LPAREN:( 
ID:f 
RPAREN:) 
ID:f 
PLUS:+ 
NUM:1 
RPAREN:) 
SEMI:; 
//:eof://:eof 
Handling definition
RPAREN:) 
Handling top level expression

Compilation failed!
ERROR: Unknown token when expecting an expression
